### PR TITLE
Update context.

### PR DESCRIPTION
**Pull request by davidlehn**
- Update dependencies.
- Update actions.
- Require Node.js 18+.
- Use JSON style for context data.
- Update to draft context.

Note that this may be a temporary branch.  This was going to be added to the `credentials-context` package, but doing so exposed a number of difficulties and it was easier to add here at the moment.

### CODE REVIEW

**Review by BigBlueHat — APPROVED** [review]
Thanks for switching to the JSON style object!

**Review by dlongley — APPROVED** [review]
Approving, but we shouldn't merge until we get the context changes that are currently in PRs on the vcdm repo.

File: contexts/credentials-v2.jsonld [hunk 1]
```diff
@@ -84,7 +147,50 @@
...
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "jsonSchema": {
+          "@id": "https://w3.org/2018/credentials#jsonSchema",
+          "@type": "@json"
+        }
+      }
+    },
+    "StatusList2021Credential": "https://w3id.org/vc/status-list#StatusList2021Credential",
```
Comment (brianorwhatever) [inline contexts/credentials-v2.jsonld]:
```suggestion
    "BitstringStatusListCredential": "https://www.w3.org/ns/credentials/status#BitstringStatusListCredential",
```

File: contexts/credentials-v2.jsonld [hunk 2]
```diff
@@ -84,7 +147,50 @@
...
+    "StatusList2021": {
```
Comment (brianorwhatever) [inline contexts/credentials-v2.jsonld]:
```suggestion
    "BitstringStatusList": {
```

File: contexts/credentials-v2.jsonld [hunk 3]
```diff
@@ -84,7 +147,50 @@
...
+      "@id": "https://w3id.org/vc/status-list#StatusList2021",
```
Comment (brianorwhatever) [inline contexts/credentials-v2.jsonld]:
```suggestion
      "@id": "https://www.w3.org/ns/credentials/status#BitstringStatusList",
```

File: contexts/credentials-v2.jsonld [hunk 4]
```diff
@@ -84,7 +147,50 @@
...
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "statusPurpose": "https://w3id.org/vc/status-list#statusPurpose",
```
Comment (brianorwhatever) [inline contexts/credentials-v2.jsonld]:
```suggestion
        "statusPurpose": "https://w3.org/ns/credentials/status#statusPurpose",
```

File: contexts/credentials-v2.jsonld [hunk 5]
```diff
@@ -84,7 +147,50 @@
...
+        "encodedList": "https://w3id.org/vc/status-list#encodedList"
```
Comment (brianorwhatever) [inline contexts/credentials-v2.jsonld]:
```suggestion
        "encodedList": "https://w3.org/ns/credentials/status#encodedList",
```

File: contexts/credentials-v2.jsonld [hunk 6]
```diff
@@ -84,7 +147,50 @@
...
+      }
```
Comment (brianorwhatever) [inline contexts/credentials-v2.jsonld:204]:
```suggestion
        "ttl": "https://www.w3.org/ns/credentials/status#ttl",
        "statusReference": "https://www.w3.org/ns/credentials/status#statusReference",
        "statusSize": "https://www.w3.org/ns/credentials/status#statusSize",
        "statusMessage": {
          "@id": "https://www.w3.org/ns/credentials/status#statusMessage",
          "@context": {
            "@protected": true,

            "id": "@id",
            "type": "@type",

            "status": "https://www.w3.org/ns/credentials/status#status",
            "message": "https://www.w3.org/ns/credentials/status#message"
          }
        }
      }
```

File: contexts/credentials-v2.jsonld [hunk 7]
```diff
@@ -84,7 +147,50 @@
...
+    },
+    "StatusList2021Entry": {
```
Comment (brianorwhatever) [inline contexts/credentials-v2.jsonld]:
```suggestion
    "BitstringStatusListEntry": {
```

File: contexts/credentials-v2.jsonld [hunk 8]
```diff
@@ -84,7 +147,50 @@
...
+      "@id": "https://w3id.org/vc/status-list#StatusList2021Entry",
```
Comment (brianorwhatever) [inline contexts/credentials-v2.jsonld]:
```suggestion
      "@id": "https://www.w3.org/ns/credentials/status#BitstringStatusListEntry",
```

File: contexts/credentials-v2.jsonld [hunk 9]
```diff
@@ -84,7 +147,50 @@
...
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "statusPurpose": "https://w3id.org/vc/status-list#statusPurpose",
```
Comment (brianorwhatever) [inline contexts/credentials-v2.jsonld]:
```suggestion
        "statusPurpose": "https://www.w3.org/ns/credentials/status#statusPurpose",
```

File: contexts/credentials-v2.jsonld [hunk 10]
```diff
@@ -84,7 +147,50 @@
...
+        "statusListIndex": "https://w3id.org/vc/status-list#statusListIndex",
```
Comment (brianorwhatever) [inline contexts/credentials-v2.jsonld]:
```suggestion
        "statusListIndex": "https://www.w3.org/ns/credentials/status#statusListIndex",
```

File: contexts/credentials-v2.jsonld [hunk 11]
```diff
@@ -84,7 +147,50 @@
...
+        "statusListCredential": {
+          "@id": "https://w3id.org/vc/status-list#statusListCredential",
```
Comment (brianorwhatever) [inline contexts/credentials-v2.jsonld]:
```suggestion
          "@id": "https://www.w3.org/ns/credentials/status#statusListCredential",
```

**Review by brianorwhatever — APPROVED** [review]
`StatusList2021` -> `BitstringStatusList`

**Comment by BigBlueHat** [comment]
@davidlehn how often do we want to do this? I think we need to get this merged sooner than later...and do it again (most likely) as often as we can to keep pace with the VC Data Model 2.0 spec.

Essentially, I'm wondering what's keeping this in draft?

**Comment by davidlehn** [comment]
@BigBlueHat:
- I don't know.
- This *could* be updated on every upstream context change.  Will that disrupt other development?  Or help it?
- Should this be released?  Maybe as 0.x.x until VCDM 2.0 is out?
- I'm not convinced this package should even exist.  A new package for every context version is a maintenance hardship.  One credentials context package could support both.  (With needed context package design updates.)

**Comment by davidlehn** [comment]
- Made a new PR for context sync, if that's what we want to do here.
- https://github.com/digitalbazaar/credentials-v2-context/pull/6
- @brianorwhatever: Closing your comments in favor of a PR for the sync change.  Easier to review as one change and there were some issues like `w3.org` vs `www.w3.org`.

**Pull request merged**